src/main/java/com/google/sps/data/UserProfile.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+public final class UserProfile{
```

Comment:
Why are we calling this user profile?

Reply:
I just noticed that I messed up with the names. I wanted use objects to convert the information into JSON by using GSON instead manually writing line by line.